cmd/locode/db/boltdb/calls.go

```diff
@@ -0,0 +1,166 @@
+package locodebolt
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+	"go.etcd.io/bbolt"
+)
+
+// Open opens an underlying BoltDB instance.
+//
+// Timeout of BoltDB opening is 3s (only for Linux or Darwin).
+//
+// Opens BoltDB in read-only mode if DB is read-only.
+func (db *DB) Open() error {
+	// copy-paste from metabase:
+	// consider universal Open/Close for BoltDB wrappers
+
+	err := os.MkdirAll(filepath.Dir(db.path), db.mode)
```

Comment:
Ah, and please drop this logic completely. Just open a DB and expect dir to be present.